src/audio/module_adapter/module/volume/volume.c

```diff
@@ -281,6 +281,7 @@ static void volume_ramp(struct processing_module *mod)
 			cd->volume[i] = cd->tvolume[i];
```

Comment:
PLease fix your signed off line. It should be `Signed-off-by: FirstName LastName <email>`.

Reply:
Sorry, not sure when it was changed, fixed.